docs/quickstarts/insights-integrating-with-satellite/metadata.yml

```
@@ -0,0 +1,17 @@
+kind: QuickStarts
+name: insights-inventory-integrating-with-satellite
```

@knortema I think the name here should be `insights-integrating-with-satellite`, right?
```suggestion
name: insights-integrating-with-satellite
```